src/app/query/stacks/balance/stx-balance.hooks.ts

```diff
@@ -23,10 +23,12 @@ export function parseBalanceResponse(balances: AddressBalanceResponse) {
     ])
   ) as Record<AccountBalanceStxKeys, Money>;
 
-  const stx: AccountStxBalanceBigNumber & { availableStx: Money } = {
+  const stx: AccountStxBalanceBigNumber & { availableStx: Money; unlockedStx: Money } = {
     ...balances.stx,
     ...stxMoney,
+    balance: createMoney(stxMoney.balance.amount, 'STX'),
     availableStx: createMoney(stxMoney.balance.amount.minus(stxMoney.locked.amount), 'STX'),
```

Comment:
Shouldn't L30 be deleted now in favor of using `unlockedStx` throughout the code?

Reply:
yes, fixed.
my first intention was to do it in separate pr, but now think its better to do it here